packages/react-graphql/src/tests/async.test.tsx

```diff
@@ -154,6 +156,7 @@ describe('createAsyncQueryComponent()', () => {
         <AsyncQueryComponent.KeepFresh pollInterval={pollInterval} />,
       );
 
+      // eslint-disable-next-line jest/prefer-strict-equal
       expect(trigger(keepFresh.find(Async), 'render', query)).toEqual(
```

Comment:
The only way I could get these to pass was by stringifying the 2 values, I figured just using `toEqual` was safe enough.